iceberg/pom.xml

```diff
@@ -247,7 +247,6 @@
         <version>${spotless.maven.plugin.version}</version>
         <configuration>
           <skip>${validate.skip}</skip>
-          <argLine>--add-exports jdk.compiler/com.sun.tools.javac.util=ALL-UNNAMED</argLine>
```

Comment:
argLine is for surefire plugin. Spotless does not fork a new JVM for its operations; it runs inside the main Maven process. As we have this add-opens in maven-compiler-plugin in iceberg/pom.xml.